PC/winreg.c

```diff
@@ -964,23 +969,36 @@ If the function fails, an OSError exception is raised.
 
 static HKEY
 winreg_CreateKeyEx_impl(PyObject *module, HKEY key, const wchar_t *sub_key,
-                        int reserved, REGSAM access)
-/*[clinic end generated code: output=51b53e38d5e00d4b input=42c2b03f98406b66]*/
+                        int reserved, REGSAM access, int options,
+                        int create_only)
+/*[clinic end generated code: output=10c0a5f7beea07e3 input=23d740b8cd7fb0df]*/
 {
     HKEY retKey;
     long rc;
+    DWORD disposition;
 
     if (PySys_Audit("winreg.CreateKey", "nun",
                     (Py_ssize_t)key, sub_key,
                     (Py_ssize_t)access) < 0) {
         return NULL;
     }
-    rc = RegCreateKeyExW(key, sub_key, reserved, NULL, 0,
-                         access, NULL, &retKey, NULL);
+    rc = RegCreateKeyExW(key, sub_key, reserved, NULL, options,
+                         access, NULL, &retKey, &disposition);
     if (rc != ERROR_SUCCESS) {
         PyErr_SetFromWindowsErrWithFunction(rc, "CreateKeyEx");
         return NULL;
     }
+    if (create_only == TRUE) {
```

Comment:
Strictly speaking, this isn't the right `TRUE`. I think we always guarantee it'll be `1`, and `TRUE` is basically guaranteed by Windows to be `1`, but let's just use a non-zero test:

```suggestion
    if (create_only) {
```